src/en/_api_endpoints.md

```diff
@@ -11,7 +11,7 @@ NotificationClient client = new NotificationClient(apiKey, "https://api.notifica
 ```vbnet
 using Notify.Client;
 
-var client = new NotificationClient(apiKey, "https://api.notification.canada.ca");
+var client = new NotificationClient("https://api.notification.canada.ca", apiKey);
```

Comment:
We'd need to fix the French version too!
https://documentation.notification.canada.ca/fr/clients.html

Reply:
Good catch!  In this case the french file makes use of the same partial .md though, so we are good 😓